packages/react/src/components/NavigationTabs/CountLabel/index.tsx

```diff
@@ -0,0 +1,83 @@
+import * as designTokens from '@project44-manifest/design-tokens';
+import { CSS } from '@project44-manifest/react-styles';
+import { Box } from '../../box';
+import { Spinner } from '../../Spinner';
+import { Typography } from '../../Typography';
+
+interface Props {
+  value: number | string;
+  variant?: 'black' | 'blue' | 'darkBlue' | 'darkGrey' | 'default' | 'grey' | 'lightGrey' | 'red';
+  dense?: boolean;
+  css?: CSS;
+  isLoading?: boolean;
+}
+
+export function CountLabel({ value, variant = 'default', dense = false, css, isLoading }: Props) {
+  const theme = () => {
+    switch (variant) {
+      case 'black':
+        return {
+          backgroundColor: designTokens.colorTextPrimary,
```

Comment:
can't we use string tokens here? like "$colors$text-primary"

Reply:
Thanks @vitaliy-p44 for pointing this, Updated code accordingly.